### ISSUE TITLE
Think of a way for the user to choose which repos to auto-import(more settings to the auto import behavior )

### ISSUE THREAD

**Issue by ramiroJCB** [mergestat/mergestat#250]
Right now if a  user wants to delete a repo from the import , it will be auto-imported on the next session.

**Comment by patrickdevivo** [comment]
I think this is more about allowing for a user to explicitly state which repos they want to _exclude_ from a repo auto import - so that certain repos are not added into fuse.

Is that what you are describing @ramiroJCB?

**Comment by mattheworiordan** [comment]
I agree, this is problematic. I have auto-imported around 300 repos, a few of which are problematic (they are forks, so skew the stats). In addition, archived repos are being imported. Having the ability to delete / exclude a repo is sorely needed. Thanks!
